Update on Mac runner build

iseeyuan opened the pull request.

larryliu0820 commented on examples/models/llama2/README.md:
```diff
@@ -195,6 +195,7 @@ The Wikitext results generated above used: `{max_seq_len: 2048, limit: 1000}`
 ## Step 4: Run on your computer to validate
 
 1. Build executorch with optimized CPU performance as follows. Build options available [here](https://github.com/pytorch/executorch/blob/main/CMakeLists.txt#L59).
+Note: For Mac users: There's a known linking issue with Xcode 15.1. Update Xcode to the most recent version to resolve the linking issue. 
```
Can you add a FAQ or something and describe the issue and the solution?

iseeyuan committed:
Update on Mac runner build (#4601)

Summary: Pull Request resolved: https://github.com/pytorch/executorch/pull/4601

Differential Revision: D60986784
